---
layout: default
title: Gu Lab
subtitle: Shengqing Stan Gu Lab
use-site-title: true
---

<div class="jumbotron">
    <div class="container">
        <div class="row">
            <div class="col-lg-3">
                <p>
                    <img src="/img/people/shirley_liu.png" width="230" height="230">
                </p>
            </div>
            <div class="col-lg-9">
                <p>Shengqing Stan Gu graduated from University of Toronto with a PhD in Medical Biophysics and did postdoctoral training at Harvard Medical School and Dana-Farber Cancer Institute.
			He joined University of Texas MD Anderson Cancer Center as Assistant Professor in November 2022.
			He is strongly committed to the professional fulfillment and career advancement of his trainees.

                </p>
            </div>
        </div>
    </div>
</div>

<div class="container">
    <div class="row">
        <h2>Research interests</h2>
        <p>Our lab is interested in enhancing the efficacy of cancer therapies, especially immunotherapy, by modulating cell-cell interactions.
		To achieve this goal, we integrate various cutting-edge experimental and computational approaches in the areas of functional genomics, single-cell/spatial multi-omics, data mining, machine learning, and mathematical modeling, among others.
		
        </p>
    </div>
    <div class="row">
        <h3>Cancer Epigenetics</h3>
        <div class="col-lg-3">
            <img src="/img/research/Cistrome.png">
        </div>
        <div class="col-lg-9">
            <p>We have been developing algorithms (MACS, Cistrome, NPS, BETA) to facilitate the analysis of epigenomic data and use integrative modeling approaches to study genomic transcriptional and epigenetic gene regulatory mechanisms underlying tumorigenesis and progression.
                We are developing new methods to utilize the abundant public ChIP-seq data to infer functional enhancers that regulate the expression, and understand the targets and specificity of epigenetic drugs in cancers.
            </p>
        </div>
    </div>
    <div class="row">
        <h3>CRISPR screens</h3>
        <div class="col-lg-3">
            <img src="/img/research/CRISPR.png">
        </div>
        <div class="col-lg-9">
            <p>
                We are developing the computational methods for the design (SSC), analysis (MAGeCK), hit prioritization (NEST), and visualization (VISPR) of genome-wide CRISPR screens.
                We are also using this technology to identify key genes in breast and prostate tumor progression and drug resistance.
                We also develop CRISPR screen platforms to understand the functions of enhancers and long-noncoding RNAs, and identify synthetic lethal gene pairs in cancer that leads to optimized cancer precision medicine.
            </p>
        </div>
    </div>
    <div class="row">
        <h3>Cancer Immunology</h3>
        <div class="col-lg-3">
            <img src="/img/research/TIMER.png">
        </div>
        <p>
            We are developing novel computational algorithms to systematically deconvolve the abundance of different immune cell types.
            Applying the deconvolution method on over 10K tumor samples from TCGA identified widespread clinical associations with tumor infiltrating immune cells.
            Our investigations on cancer vaccine and checkpoint blockade targets suggest that systematic analyses of tumor immunity have the potential to inform effective cancer immunotherapies.
        </p>
    </div>
    <div class="row">
        <h3>Single Cell</h3>
        <div class="col-lg-3">
            <img src="/img/research/MAESTRO.png">
        </div>
        <p>
            We are developing computational methods (<a href="http://github.com/liulab-dfci/MAESTRO">MAESTRO</a>) for integrative analysis of single-cell RNA-seq (scRNA-seq) and ATAC-seq (scATAC-seq) data. By integrating the public transcription factor (TF) ChIP-seq datasets in Cistrome DB, we are unraveling the regulatory network with tumor microenvironment and identifying key regulators between different immune cells. We are also developing novel computational algorithms for spatial transcriptomic data analysis.
        </p>
    </div>

</div>

<!--<div class="posts-list">-->
  <!--{% for post in paginator.posts %}-->
  <!--<article class="post-preview">-->
    <!--<a href="{{ post.url | relative_url }}">-->
	  <!--<h2 class="post-title">{{ post.title }}</h2>-->

	  <!--{% if post.subtitle %}-->
	  <!--<h3 class="post-subtitle">-->
	    <!--{{ post.subtitle }}-->
	  <!--</h3>-->
	  <!--{% endif %}-->
    <!--</a>-->

    <!--<p class="post-meta">-->
      <!--Posted on {{ post.date | date: site.date_format }}-->
    <!--</p>-->

    <!--<div class="post-entry-container">-->
      <!--{% if post.image %}-->
      <!--<div class="post-image">-->
        <!--<a href="{{ post.url | relative_url }}">-->
          <!--<img src="{{ post.image | relative_url }}">-->
        <!--</a>-->
      <!--</div>-->
      <!--{% endif %}-->
      <!--<div class="post-entry">-->
        <!--{{ post.excerpt | strip_html | xml_escape | truncatewords: site.excerpt_length }}-->
        <!--{% assign excerpt_word_count = post.excerpt | number_of_words %}-->
        <!--{% if post.content != post.excerpt or excerpt_word_count > site.excerpt_length %}-->
          <!--<a href="{{ post.url | relative_url }}" class="post-read-more">[Read&nbsp;More]</a>-->
        <!--{% endif %}-->
      <!--</div>-->
    <!--</div>-->

    <!--{% if post.tags.size > 0 %}-->
    <!--<div class="blog-tags">-->
      <!--Tags:-->
      <!--{% if site.link-tags %}-->
      <!--{% for tag in post.tags %}-->
      <!--<a href="{{ '/tags' | relative_url }}#{{- tag -}}">{{- tag -}}</a>-->
      <!--{% endfor %}-->
      <!--{% else %}-->
        <!--{{ post.tags | join: ", " }}-->
      <!--{% endif %}-->
    <!--</div>-->
    <!--{% endif %}-->

   <!--</article>-->
  <!--{% endfor %}-->
<!--</div>-->

<!--{% if paginator.total_pages > 1 %}-->
<!--<ul class="pager main-pager">-->
  <!--{% if paginator.previous_page %}-->
  <!--<li class="previous">-->
    <!--<a href="{{ paginator.previous_page_path | relative_url }}">&larr; Newer Posts</a>-->
  <!--</li>-->
  <!--{% endif %}-->
  <!--{% if paginator.next_page %}-->
  <!--<li class="next">-->
    <!--<a href="{{ paginator.next_page_path | relative_url }}">Older Posts &rarr;</a>-->
  <!--</li>-->
  <!--{% endif %}-->
<!--</ul>-->
<!--{% endif %}-->
