---
layout: default
title: Gu Lab
subtitle: Shengqing Stan Gu Lab
use-site-title: true
---

<div class="jumbotron">
    <div class="container">
        <div class="row">
            <div class="col-lg-3">
                <p>
                    <img src="/img/people/shirley_liu.png" width="230" height="230">
                </p>
            </div>
            <div class="col-lg-9">
                <p>Shengqing Stan Gu graduated from University of Toronto with a PhD in Medical Biophysics and did postdoctoral training at Harvard Medical School and Dana-Farber Cancer Institute.
			He joined University of Texas MD Anderson Cancer Center as Assistant Professor in November 2022.
			He is strongly committed to the professional fulfillment and career advancement of his trainees.

                </p>
            </div>
        </div>
    </div>
</div>

<div class="container">
    <div class="row">
        <h2>Research interests</h2>
        <p>Our lab is interested in enhancing the efficacy of cancer therapies, especially immunotherapy, by modulating cell-cell interactions.
		To achieve this goal, we integrate various cutting-edge experimental and computational approaches in the areas of functional genomics, single-cell/spatial multi-omics, data mining, machine learning, and mathematical modeling, among others.
		
        </p>
    </div>
    <div class="row">
        <h3>Context-dependent gene regulation</h3>
        <div class="col-lg-3">
            <img src="/img/research/Cistrome.png">
        </div>
        <div class="col-lg-9">
            <p>
		    We aim to perturb gene expression in the tumor to enhance immunotherapy with desired immunomodulatory effects.
		    Our previous research revealed that gene expression regulation depends on the cellular genetic/epigenetic context.
		    For example, SHP2 can regulate different downstream pathways between WT and BCR-ABL1+ pre-B cells; cancer cells with different baseline NF-κB expression can show different responses to SMAC mimetics in upregulating MHC-I.
		    Therefore, to achieve the desired immunomodulatory effects, we need rational, personalized strategies to perturb gene expression.
		    To this end, we will integrate functional genomics, single-cell/spatial technologies, and machine learning approaches to elucidate the context-dependent gene regulation mechanisms.
		    
            </p>
        </div>
    </div>
    <div class="row">
        <h3>Novel mechanisms of cell-cell interaction</h3>
        <div class="col-lg-3">
            <img src="/img/research/CRISPR.png">
        </div>
        <div class="col-lg-9">
            <p>
		    The essence of the cancer immune response lies in cell-cell interactions.
		    Our previous research investigated multiple aspects of such interaction at different levels.
		    We applied machine learning models to public bulk tumor gene expression data and identified SERPINB9 and MAN2A1 as regulators of cancer cells’ response to cytotoxic T cells.
		    Using in vivo CRISPR screens, we identified a ubiquitin E3 ligase, COP1, whose inhibition can alter macrophage infiltration and sensitize tumors to immunotherapy.
		    We also identified multiple regulators of the MHC-I/II antigen presentation machinery in cancer cells that can modulate cancer-T-cell interaction.
		    Of special interest is the function and regulation of MHC-II in cancer cells, which is poorly understood and has great translational potential.
		    Furthermore, we are interested in other cell-cell interaction mechanisms in the tumor microenvironment.
            </p>
        </div>
    </div>
    <div class="row">
        <h3>Cancer Immunology</h3>
        <div class="col-lg-3">
            <img src="/img/research/TIMER.png">
        </div>
        <p>
            We are developing novel computational algorithms to systematically deconvolve the abundance of different immune cell types.
            Applying the deconvolution method on over 10K tumor samples from TCGA identified widespread clinical associations with tumor infiltrating immune cells.
            Our investigations on cancer vaccine and checkpoint blockade targets suggest that systematic analyses of tumor immunity have the potential to inform effective cancer immunotherapies.
        </p>
    </div>
    <div class="row">
        <h3>Single Cell</h3>
        <div class="col-lg-3">
            <img src="/img/research/MAESTRO.png">
        </div>
        <p>
            We are developing computational methods (<a href="http://github.com/liulab-dfci/MAESTRO">MAESTRO</a>) for integrative analysis of single-cell RNA-seq (scRNA-seq) and ATAC-seq (scATAC-seq) data. By integrating the public transcription factor (TF) ChIP-seq datasets in Cistrome DB, we are unraveling the regulatory network with tumor microenvironment and identifying key regulators between different immune cells. We are also developing novel computational algorithms for spatial transcriptomic data analysis.
        </p>
    </div>

</div>

<!--<div class="posts-list">-->
  <!--{% for post in paginator.posts %}-->
  <!--<article class="post-preview">-->
    <!--<a href="{{ post.url | relative_url }}">-->
	  <!--<h2 class="post-title">{{ post.title }}</h2>-->

	  <!--{% if post.subtitle %}-->
	  <!--<h3 class="post-subtitle">-->
	    <!--{{ post.subtitle }}-->
	  <!--</h3>-->
	  <!--{% endif %}-->
    <!--</a>-->

    <!--<p class="post-meta">-->
      <!--Posted on {{ post.date | date: site.date_format }}-->
    <!--</p>-->

    <!--<div class="post-entry-container">-->
      <!--{% if post.image %}-->
      <!--<div class="post-image">-->
        <!--<a href="{{ post.url | relative_url }}">-->
          <!--<img src="{{ post.image | relative_url }}">-->
        <!--</a>-->
      <!--</div>-->
      <!--{% endif %}-->
      <!--<div class="post-entry">-->
        <!--{{ post.excerpt | strip_html | xml_escape | truncatewords: site.excerpt_length }}-->
        <!--{% assign excerpt_word_count = post.excerpt | number_of_words %}-->
        <!--{% if post.content != post.excerpt or excerpt_word_count > site.excerpt_length %}-->
          <!--<a href="{{ post.url | relative_url }}" class="post-read-more">[Read&nbsp;More]</a>-->
        <!--{% endif %}-->
      <!--</div>-->
    <!--</div>-->

    <!--{% if post.tags.size > 0 %}-->
    <!--<div class="blog-tags">-->
      <!--Tags:-->
      <!--{% if site.link-tags %}-->
      <!--{% for tag in post.tags %}-->
      <!--<a href="{{ '/tags' | relative_url }}#{{- tag -}}">{{- tag -}}</a>-->
      <!--{% endfor %}-->
      <!--{% else %}-->
        <!--{{ post.tags | join: ", " }}-->
      <!--{% endif %}-->
    <!--</div>-->
    <!--{% endif %}-->

   <!--</article>-->
  <!--{% endfor %}-->
<!--</div>-->

<!--{% if paginator.total_pages > 1 %}-->
<!--<ul class="pager main-pager">-->
  <!--{% if paginator.previous_page %}-->
  <!--<li class="previous">-->
    <!--<a href="{{ paginator.previous_page_path | relative_url }}">&larr; Newer Posts</a>-->
  <!--</li>-->
  <!--{% endif %}-->
  <!--{% if paginator.next_page %}-->
  <!--<li class="next">-->
    <!--<a href="{{ paginator.next_page_path | relative_url }}">Older Posts &rarr;</a>-->
  <!--</li>-->
  <!--{% endif %}-->
<!--</ul>-->
<!--{% endif %}-->
